server/JsDbg.Stdio/Program.cs

```diff
@@ -16,8 +16,8 @@ class Program
 
             string extensionsDirectory = args[0];
 
-            // Inversion of control: Assume that this process has been started by python, stdio talks back to python and can ask it to do things.
-            GdbDebugger debugger = new GdbDebugger();
+            // This process has been started by python, stdio talks back to python and can ask it to do things.

```

Comment:
I know this comment has been around for a while, but can we update it to be more specific than "to do things"?